canonical-web-and-design/maas-ui
# fix(ui): add a cancel button to the script upload form

## Done

- Add a cancel button to the form for uploading commissioning and testing scripts in the settings.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the commissioning scripts upload form in the settings. There should be a cancel button.
- Clicking the button should take you back to the commissioning scripts list.
- Repeat the above steps for the testing scripts form.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1782.
